src/PrestaShopBundle/Security/OAuth2/Repository/NullScopeRepository.php

```diff
@@ -42,7 +42,7 @@ class NullScopeRepository implements ScopeRepositoryInterface
 {
     public function getScopeEntityByIdentifier($identifier): ?ScopeEntityInterface
     {
-        return null;
+        return new ScopeEntity('hook:read');
```

Comment:
I don't understand why we mention hook:read here

Reply:
When we looked into this it seemed to be required to be able to have the scopes into the token.

Reply:
maybe we can add scopes in yaml config ?

Reply:
Shouldn't we create the scope based on the parameter instead?
```suggestion
        return new ScopeEntity($identifier);
```

For now we only return it as is and consider any asked scope exists, but later we'll have to check its existence based on the scopes that were parsed from the API endpoints definitions